src/main/resources/application.properties

```diff
@@ -1,31 +1,96 @@
 # Properties file
 app.propDir=file:${catalina.base}/webapps
-spring.config.import=optional:classpath:env/${fr.insee.pogues.env:dev}/pogues-bo.properties,optional:${app.propDir}/pogues-bo.properties,optional:${app.propDir}/rmes-pogbo.properties
+
+
+# SSL: old properties, add to be false now
+fr.insee.pogues.force.ssl=false
+
+# Swagger
+fr.insee.pogues.api.host=localhost:8080
+fr.insee.pogues.api.name=
+fr.insee.pogues.api.scheme=http
+
+# Default value for cors filter : empty means no CORS allowed :
+fr.insee.pogues.cors.allowedOrigin=*
```

Comment:
in production we should use something more specific here (pogues-ui url)

An empty default value would enforce the overloading of this property which would be healthier from a security point of view

```
# CORS filter: empty means no CORS allowed, use * to allow all origins (not recommended)
fr.insee.pogues.cors.allowedOrigin=*
```